sysutils/portfile-gen/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        macports macports-contrib pypi2port-20230621
```

Comment:
This tag does technically point to the same commit as the currently most recent change to portfile-gen, but it might be less confusing to just use the commit (as future updates are unlikely to coincide with pypi2port release tags).

Also should use `github.tarball_from archive`.

Reply:
oh right I forgot that commit hashes were valid to use as version numbers; so it'd be `5447906b1c3be5b6189e61c6256dc75c291c59e5` in this case?

Reply:
ok, changed to the hash in [cba79b7](https://github.com/macports/macports-ports/pull/20951/commits/cba79b7f4ec177d2b63630c0a4e76747d41106ca).